Docker build on macOS fails with linking failure with rocksDB

the reference dockerfile -> https://github.com/status-im/nimbus-eth1/blob/master/Dockerfile

```bash
335.4 CC: nimbus_execution_client.nim
396.6 Hint:  [Link]
441.3 /usr/bin/ld: /tmp/cc524q83.ltrans122.ltrans.o: in function `aristo_rocksdb::newRocksDbCoreDbRef(string, opts::DbOptions)':
441.4 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/cache.nim:11:(.text+0xecc): undefined reference to `rocksdb_cache_create_lru'
441.4 /usr/bin/ld: /tmp/cc524q83.ltrans122.ltrans.o: in function `aristo_rocksdb::newRocksDbCoreDbRef(string, opts::DbOptions)':
441.4 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:25:(.text+0xef4): undefined reference to `rocksdb_options_create'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:37:(.text+0xf34): undefined reference to `rocksdb_options_increase_parallelism'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0xf44): undefined reference to `rocksdb_options_set_create_if_missing'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0xf54): undefined reference to `rocksdb_options_set_create_missing_column_families'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0xf64): undefined reference to `rocksdb_options_set_bytes_per_sync'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0xf80): undefined reference to `rocksdb_options_set_max_open_files'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0xf90): undefined reference to `rocksdb_options_set_allow_concurrent_memtable_write'
441.4 /usr/bin/ld: /tmp/cc524q83.ltrans122.ltrans.o: in function `aristo_rocksdb::newRocksDbCoreDbRef(string, opts::DbOptions)':
441.4 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/cache.nim:11:(.text+0xfb8): undefined reference to `rocksdb_cache_create_lru'
441.4 /usr/bin/ld: /tmp/cc524q83.ltrans122.ltrans.o: in function `aristo_rocksdb::newRocksDbCoreDbRef(string, opts::DbOptions)':
441.4 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:101:(.text+0x1058): undefined reference to `rocksdb_options_set_row_cache'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0x10c8): undefined reference to `rocksdb_options_set_max_total_wal_size'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0x10d8): undefined reference to `rocksdb_options_set_keep_log_file_num'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:48:(.text+0x10e4): undefined reference to `rocksdb_options_get_max_background_jobs'
441.4 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0x10f8): undefined reference to `rocksdb_options_set_max_subcompactions'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::close(ref<rocksdb_desc::SharedWriteBatchRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:97:(.text+0x5cb8): undefined reference to `rocksdb_writebatch_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::listColumnFamilies(string)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:25:(.text+0x944c): undefined reference to `rocksdb_options_create'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:37:(.text+0x9490): undefined reference to `rocksdb_options_increase_parallelism'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0x94a4): undefined reference to `rocksdb_options_set_create_if_missing'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0x94b4): undefined reference to `rocksdb_options_set_create_missing_column_families'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:44:(.text+0x94c8): undefined reference to `rocksdb_options_set_bytes_per_sync'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::listColumnFamilies(string)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:80:(.text+0x94f8): undefined reference to `rocksdb_list_column_families'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::listColumnFamilies(string)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:33:(.text+0x952c): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::listColumnFamilies(string)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:124:(.text+0x95c0): undefined reference to `rocksdb_options_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::listColumnFamilies(string)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/cache.nim:22:(.text+0x95e4): undefined reference to `rocksdb_cache_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::listColumnFamilies(string)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:90:(.text+0x9784): undefined reference to `rocksdb_list_column_families_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::listColumnFamilies(string)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:124:(.text+0x9794): undefined reference to `rocksdb_options_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::listColumnFamilies(string)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/cache.nim:22:(.text+0x97b8): undefined reference to `rocksdb_cache_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::get(ref<rocksdb::RocksDbRefcolonObjectType_>, openArray<uInt8>, proc<openArray<uInt8> >, ref<cfhandle::ColFamilyHandleRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:249:(.text+0x9874): undefined reference to `rocksdb_get_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::get(ref<rocksdb::RocksDbRefcolonObjectType_>, openArray<uInt8>, proc<openArray<uInt8> >, ref<cfhandle::ColFamilyHandleRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:39:(.text+0x98a0): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::get(ref<rocksdb::RocksDbRefcolonObjectType_>, openArray<uInt8>, proc<openArray<uInt8> >, ref<cfhandle::ColFamilyHandleRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:265:(.text+0x9900): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfhandle.nim:32:(.text+0x9a64): undefined reference to `rocksdb_column_family_handle_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/writeopts.nim:62:(.text+0x9b18): undefined reference to `rocksdb_writeoptions_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:662:(.text+0x9b28): undefined reference to `rocksdb_ingestexternalfileoptions_destroy'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:665:(.text+0x9b34): undefined reference to `rocksdb_flushoptions_destroy'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:650:(.text+0x9bc0): undefined reference to `rocksdb_close'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:124:(.text+0x9be4): undefined reference to `rocksdb_options_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/cache.nim:22:(.text+0x9c04): undefined reference to `rocksdb_cache_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/readopts.nim:69:(.text+0x9c28): undefined reference to `rocksdb_readoptions_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:70:(.text+0x9c80): undefined reference to `rocksdb_options_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:116:(.text+0x9ca0): undefined reference to `rocksdb_block_based_options_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb::close(ref<rocksdb::RocksDbRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/cache.nim:22:(.text+0x9cc0): undefined reference to `rocksdb_cache_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/readopts.nim:24:(.text+0xb0c0): undefined reference to `rocksdb_readoptions_create'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/writeopts.nim:22:(.text+0xb0dc): undefined reference to `rocksdb_writeoptions_create'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:59:(.text+0xb938): undefined reference to `rocksdb_options_create'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:59:(.text+0xba8c): undefined reference to `rocksdb_options_create'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:125:(.text+0xbd8c): undefined reference to `rocksdb_open_column_families'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/dbopts.nim:124:(.text+0xbdb0): undefined reference to `rocksdb_options_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/cache.nim:22:(.text+0xbdd4): undefined reference to `rocksdb_cache_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/readopts.nim:69:(.text+0xbdec): undefined reference to `rocksdb_readoptions_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/writeopts.nim:62:(.text+0xbe04): undefined reference to `rocksdb_writeoptions_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:70:(.text+0xbe54): undefined reference to `rocksdb_options_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:116:(.text+0xbe74): undefined reference to `rocksdb_block_based_options_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::open(typeDesc<ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_> >, string, ref<dbopts::DbOptionsRefcolonObjectType_>, openArray<tuple<string, ref<cfopts::ColFamilyOptionsRefcolonObjectType_> > >)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/cache.nim:22:(.text+0xbe94): undefined reference to `rocksdb_cache_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:33:(.text+0xbed4): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:37:(.text+0xbf8c): undefined reference to `rocksdb_flushoptions_create'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:141:(.text+0xbf98): undefined reference to `rocksdb_flushoptions_set_wait'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:154:(.text+0xc2ac): undefined reference to `rocksdb_ingestexternalfileoptions_create'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `putBegFn::colonanonymous__dbZkvtZkvt95initZrocks95db_u108_':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:28:(.text+0xcd18): undefined reference to `rocksdb_writebatch_create'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `putKvpFn::colonanonymous_(ref<init_common::PutHdlRefcolonObjectType_>, openArray<uInt8>, openArray<uInt8>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:70:(.text+0xcf40): undefined reference to `rocksdb_writebatch_delete_cf'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:54:(.text+0xcfd8): undefined reference to `rocksdb_writebatch_put_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `putKvpFn::colonanonymous__dbZkvt95cf_u859_(ref<init_common::PutHdlRefcolonObjectType_>, openArray<uInt8>, openArray<uInt8>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:70:(.text+0xd114): undefined reference to `rocksdb_writebatch_delete_cf'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:54:(.text+0xd1ac): undefined reference to `rocksdb_writebatch_put_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::commit(ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_>, ref<rocksdb_desc::SharedWriteBatchRefcolonObjectType_>, columnfamily::ColFamilyReadWrite, bool)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:541:(.text+0xe6b0): undefined reference to `rocksdb_write'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:39:(.text+0xe6d4): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rocksdb_desc::commit(ref<rocksdb_desc::RocksDbInstanceRefcolonObjectType_>, ref<rocksdb_desc::SharedWriteBatchRefcolonObjectType_>, columnfamily::ColFamilyReadWrite, bool)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:628:(.text+0xe9c4): undefined reference to `rocksdb_flush_cfs'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:39:(.text+0xe9e8): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rdb_put::commit_sKvtGeneric(var<rdb_desc::RdbInst>, ref<rocksdb_desc::SharedWriteBatchRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:97:(.text+0xed40): undefined reference to `rocksdb_writebatch_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `rdb_put::commit_sKvtSynchro(var<rdb_desc::RdbInst>, ref<rocksdb_desc::SharedWriteBatchRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:97:(.text+0xef44): undefined reference to `rocksdb_writebatch_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `getKvpFn::colonanonymous_(openArray<uInt8>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:249:(.text+0xf0a4): undefined reference to `rocksdb_get_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:39:(.text+0xf0d0): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:265:(.text+0xf29c): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `lenKvpFn::colonanonymous_(openArray<uInt8>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:249:(.text+0xf5dc): undefined reference to `rocksdb_get_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:39:(.text+0xf604): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:265:(.text+0xf69c): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `putEndFn::colonanonymous_(ref<init_common::PutHdlRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:97:(.text+0xfaa8): undefined reference to `rocksdb_writebatch_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `getKvpFn::colonanonymous__dbZkvt95cf_u32_(openArray<uInt8>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:249:(.text+0xff84): undefined reference to `rocksdb_get_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:39:(.text+0xffb0): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:265:(.text+0x1017c): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `lenKvpFn::colonanonymous__dbZkvt95cf_u648_(openArray<uInt8>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:249:(.text+0x104bc): undefined reference to `rocksdb_get_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:39:(.text+0x104e4): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:265:(.text+0x1057c): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans104.ltrans.o: in function `putEndFn::colonanonymous__dbZkvt95cf_u1060_(ref<init_common::PutHdlRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:97:(.text+0x10988): undefined reference to `rocksdb_writebatch_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o: in function `rdb_put::commit(var<rdb_desc::RdbInst>, ref<rocksdb_desc::SharedWriteBatchRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:97:(.text+0xeba4): undefined reference to `rocksdb_writebatch_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o: in function `getLstFn::colonanonymous_':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:249:(.text+0x14f2c): undefined reference to `rocksdb_get_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/internal/utils.nim:39:(.text+0x14f58): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o:/root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/rocksdb.nim:265:(.text+0x151c8): undefined reference to `rocksdb_free'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o: in function `putBegFn::colonanonymous_':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:28:(.text+0x153e8): undefined reference to `rocksdb_writebatch_create'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o: in function `putLstFn::colonanonymous_(ref<desc_backend::PutHdlRefcolonObjectType_>, desc_structural::SavedState)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:54:(.text+0x156e8): undefined reference to `rocksdb_writebatch_put_cf'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:70:(.text+0x15748): undefined reference to `rocksdb_writebatch_delete_cf'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o: in function `putEndFn::colonanonymous_(ref<desc_backend::PutHdlRefcolonObjectType_>)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:97:(.text+0x15924): undefined reference to `rocksdb_writebatch_destroy'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o: in function `aristo_rocksdb::toCfOpts(opts::DbOptions, ref<cache::CacheRefcolonObjectType_>, bool)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:52:(.text+0x15978): undefined reference to `rocksdb_block_based_options_create'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x1598c): undefined reference to `rocksdb_block_based_options_set_block_size'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x1599c): undefined reference to `rocksdb_block_based_options_set_cache_index_and_filter_blocks'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x159ac): undefined reference to `rocksdb_block_based_options_set_pin_l0_filter_and_index_blocks_in_cache'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:36:(.text+0x159d0): undefined reference to `rocksdb_filterpolicy_create_ribbon_hybrid'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:102:(.text+0x159e4): undefined reference to `rocksdb_block_based_options_set_filter_policy'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:90:(.text+0x15a48): undefined reference to `rocksdb_block_based_options_set_block_cache'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x15a8c): undefined reference to `rocksdb_block_based_options_set_index_type'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x15a9c): undefined reference to `rocksdb_block_based_options_set_pin_top_level_index_and_filter'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x15aac): undefined reference to `rocksdb_block_based_options_set_cache_index_and_filter_blocks_with_high_priority'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x15abc): undefined reference to `rocksdb_block_based_options_set_partition_filters'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x15acc): undefined reference to `rocksdb_block_based_options_set_data_block_index_type'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/options/tableopts.nim:64:(.text+0x15adc): undefined reference to `rocksdb_block_based_options_set_data_block_hash_ratio'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o: in function `aristo_rocksdb::toCfOpts(opts::DbOptions, ref<cache::CacheRefcolonObjectType_>, bool)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:59:(.text+0x15af4): undefined reference to `rocksdb_options_create'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:157:(.text+0x15bc8): undefined reference to `rocksdb_options_set_block_based_table_factory'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15c14): undefined reference to `rocksdb_options_set_write_buffer_size'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:180:(.text+0x15c28): undefined reference to `rocksdb_options_set_memtable_whole_key_filtering'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15c3c): undefined reference to `rocksdb_options_set_memtable_prefix_bloom_size_ratio'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15c4c): undefined reference to `rocksdb_options_set_level0_file_num_compaction_trigger'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15c5c): undefined reference to `rocksdb_options_set_bottommost_compression'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:82:(.text+0x15c68): undefined reference to `rocksdb_options_get_write_buffer_size'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:176:(.text+0x15cc0): undefined reference to `rocksdb_options_set_memtable_vector_rep'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15d20): undefined reference to `rocksdb_options_set_max_bytes_for_level_base'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15d30): undefined reference to `rocksdb_options_set_max_bytes_for_level_multiplier'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:82:(.text+0x15d3c): undefined reference to `rocksdb_options_get_write_buffer_size'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15d50): undefined reference to `rocksdb_options_set_target_file_size_base'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15d60): undefined reference to `rocksdb_options_set_target_file_size_multiplier'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15d70): undefined reference to `rocksdb_options_set_ttl'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/columnfamily/cfopts.nim:78:(.text+0x15d80): undefined reference to `rocksdb_options_set_periodic_compaction_seconds'
441.5 /usr/bin/ld: /tmp/cc524q83.ltrans121.ltrans.o: in function `rdb_put::putVtx(var<rdb_desc::RdbInst>, ref<rocksdb_desc::SharedWriteBatchRefcolonObjectType_>, tuple<desc_identifiers::VertexID, desc_identifiers::VertexID>, ref<desc_structural::VertexRefcolonObjectType_>, desc_identifiers::HashKey)':
441.5 /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:54:(.text+0x1b708): undefined reference to `rocksdb_writebatch_put_cf'
441.5 /usr/bin/ld: /root/nimbus-eth1/vendor/nim-rocksdb/rocksdb/writebatch.nim:70:(.text+0x1b88c): undefined reference to `rocksdb_writebatch_delete_cf'
441.7 collect2: error: ld returned 1 exit status
441.7 Error: execution of an external program failed: 'g++  @nimbus_execution_client_linkerArgs.txt'
441.8 make: *** [Makefile:219: nimbus_execution_client] Error 1
------
Dockerfile:22
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,5 @@
 # Nimbus
-# Copyright (c) 2024-2025 Status Research & Development GmbH
+# Copyright (c) 2024-2026 Status Research & Development GmbH
 # Licensed under either of
 #  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or
 #    http://www.apache.org/licenses/LICENSE-2.0)

